ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-edge-cases-test.js

```diff
@@ -309,6 +319,19 @@ module('Acceptance | kv-v2 workflow | edge cases', function (hooks) {
     assert.false(codemirror().getValue().includes('*'), 'Values are not obscured on edit view');
   });
 
+  test('on enter the JSON editor cursor goes to the next line', async function (assert) {
```

Comment:
I confirmed that if I run this test on main currently, it fails.

Reply:
thank you for confirming!